tests/oracle/test-oracle.cpp

```diff
@@ -85,6 +93,51 @@ TEST_CASE("Oracle datetime", "[oracle][datetime]")
 
         CHECK(t_out == std::string(buf));
     }
+
+    {
+        // date and time - between years 1- 2201
+        soci::session sql(backEnd, connectString);
+
+        table_creator_for_timestamp tableCreator(sql);
+
+        for(int i = 1; i <= 2201; i = i + 50)
+        {
+            char t[10];
+            sprintf(t, "%04d", i);
+
+            std::string date = std::to_string(i) + "-03-28 14:06:13";
+            std::tm t1 {}, t2 {}, t4 {};
+            strptime(date.c_str(), "%Y-%m-%d %H:%M:%S", &t2);
```

Comment:
I don't think we can use `strptime()` under Windows, unfortunately.

Reply:
Changed to std::get_time, unfortunately it doesn't work well with years < 100, so I changed the tested years range also.
Tested on Windows and Ubuntu.